src/pages/index.md

```diff
@@ -1,10 +1,52 @@
 ---
-title: Overview - Adobe Commerce Cloud Tools
+title: Commerce Cloud Tools
 description: This is the overview page of the Commerce Cloud Tools.
 ---
 
-# Adobe Commerce Cloud Tools
+<Hero slots="image, heading, text"/>
 
-## Contributing
+![Commerce Cloud Docker](_images/banner-hex-violet.png)
 
-We encourage you to participate in our open documentation initiative, if you have suggestions, corrections, additions or deletions for this documentation, check out the source from [this github repo](https://github.com/adobe/gatsby-theme-spectrum-example), and submit a pull request with your contribution.
+# Commerce Cloud Tools
+
+This is the overview page of the Commerce Cloud Tools.
+
+<Resources slots="heading, links"/>
+
+#### Resources
+
+- [Adobe Commerce Developer Documentation](https://developer.adobe.com/commerce/docs) (Adobe Developer site)
+- [Adobe Commerce Documentation](https://experienceleague.adobe.com/docs/commerce.html) (Adobe Experience League site)
+
+## Overview
+
+This documentation provides resources for deploying Adobe Commerce to a Docker environment for development, testing, and automation tasks.
+
+## Discover
+
+<DiscoverBlock width="100%" slots="heading, link, text"/>
+
+### Cloud Infrastructure
+
+[Commerce on Cloud Infrastructure guide](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/overview.html)
+
+Learn more about our automated hosting platform with a self-service approach to building, deploying, and managing your Commerce application in a cloud-native environment.
+
+<DiscoverBlock slots="heading, link, text"/>
+
+### Tools
+
+[Cloud Docker for Commerce](/docker/)
+
+Use the Cloud Docker tool for development and testing.
+
+<DiscoverBlock width="100%" slots="heading, text"/>
+
+### Commerce documentation
```

Comment:
I would either remove this since its a duplicate of the "Resources" links on the right (line 16) or delete the resources block. I don't think you need both. If you delete the resource block, feel free to ignore my suggestion to remove the "Cloud infrastructure" section under the Discover block.